modules/Hoot/ui/managePanel/transAssist/upload.js

```diff
@@ -51,46 +55,75 @@ export default class Upload {
         this.createUploadForm();
         this.createSchemaSelector();
         this.createUploadButtons();
+        this.loadSchemas();
     }
 
     createUploadForm() {
         this.uploadForm = this.instance.panelWrapper
             .append( 'form' )
             .classed( 'ta-upload-form round keyline-all fill-white', true );
     }
+    async loadSchemas() {
```

Comment:
I don't think we need this function as it just swaps this.schemaOptions over to this.schemaList.